kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver_with_constraints.h

```diff
@@ -1186,7 +1232,53 @@ class ResidualBasedEliminationBuilderAndSolverWithConstraints
                     ConstructRelationMatrixStructure(pScheme, rTMatrix, rModelPart);
                 }
             }
+
+            // Resizing the system vector
+            if (mpConstantVector->size() != BaseType::mEquationSystemSize || BaseType::GetReshapeMatrixFlag() || mCleared) {
+                mpConstantVector->resize(BaseType::mEquationSystemSize, false);
+                mComputeConstantContribution = ComputeConstraintContribution(pScheme, rModelPart);
+            } else {
+                if (mpConstantVector->size() != BaseType::mEquationSystemSize) {
+                    KRATOS_ERROR <<"The equation system size has changed during the simulation. This is not permited."<<std::endl;
+                    mpConstantVector->resize(BaseType::mEquationSystemSize, false);
+                    mComputeConstantContribution = ComputeConstraintContribution(pScheme, rModelPart);
+                }
+            }
+        }
+    }
+
+    /**
+     * @brief It computes the reactions of the system
+     * @param pScheme The pointer to the integration scheme
+     * @param rModelPart The model part to compute
+     * @param rA The LHS matrix of the system of equations
+     * @param rDx The vector of unkowns
+     * @param rb The RHS vector of the system of equations
+     */
+    void CalculateReactions(
+        typename TSchemeType::Pointer pScheme,
+        ModelPart& rModelPart,
+        TSystemMatrixType& rA,
+        TSystemVectorType& rDx,
+        TSystemVectorType& rb
+        ) override
+    {
+        KRATOS_TRY
+
+        // Refresh RHS to have the correct reactions
+        BuildRHS(pScheme, rModelPart, rb);
+
+        // Adding contribution to reactions
+        TSystemVectorType& r_reactions_vector = *BaseType::mpReactionsVector;
+
+        // Updating variables
+        for (auto& r_dof : BaseType::mDofSet) {
+            if ((r_dof.IsFixed()) || mDoFSlaveSet.find(r_dof) != mDoFSlaveSet.end()) {
+                r_dof.GetSolutionStepReactionValue() = -r_reactions_vector[mReactionEquationIdMap[r_dof.EquationId()]];
```

Comment:
The reactions are not set on all over the domain ?? why not ? why only on the fixed and on the slaves ? 
@RiccardoRossi  I remember we said they should be output all over the domain ??

Reply:
I understood the complete opposite

Reply:
i think it is better to compute the reactions all over (which is what i was telling you over WhatsApp), since a non zero residual in the interior is a way of observing the error

Reply:
Yes, but can we change that all together in all BS in a future PR?

Reply:
The block versions already do it like that. !
When we say we do it like that in a future PR, we can remove this calculation in this B&S or ?(may be later)

Reply:
let me give some pseudocode for elements. Let's assume that our database is ok (both on masters and slaves) since this is what happens when you finish the Solve

(the next is pseudocode)

~~~cpp
... here set to zero the reactions on all the dofs ...

//here compute the reactions by assembling the elemental (and conditional) contributions
for(auto elem : mrModelPart.Elements)
      RHS = elem.CalculateRHS(...)
      el_dofs = elem.GetDofList()
      for(unsigned int i = 0; i<RHS.size(); ++i)
            el_dofs[i].GetSolutionStepReactionValue() -= RHS[i]



~~~